capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -228,7 +233,7 @@ function c4m_restful_quick_post_preprocess_html(&$vars) {
 function c4m_restful_quick_post_needs_app() {
   // Check first if the user is on the global homepage.
   // Or if he is on a group dashboard he has no access to.
-  if (drupal_is_front_page() && !og_context()) {
+  if (drupal_is_front_page() && !empty(og_purl_og_context_handler()) && !og_context()) {
```

Comment:
We can remove this check from now: from the moment we are on the <front> page we need the app.

See [Line 236](https://github.com/capacity4dev/capacity4more/pull/458/files#diff-8f3e84672c8f835559f9eb60f7e32eecL236)